packages/cli/test/unit/commands/teams/index.unit.test.ts

```diff
@@ -0,0 +1,24 @@
+import {stdout, stderr} from 'stdout-stderr'
+import Cmd  from '../../../../src/commands/teams'
+import runCommand from '../../../helpers/runCommand'
+import {teams} from '../../../helpers/stubs/get'
+import * as nock from 'nock'
+import expectOutput from '../../../helpers/utils/expectOutput'
+
+describe('heroku teams', () => {
+  afterEach(() => nock.cleanAll())
+
+  it('shows the teams you are a member of', async () => {
+    teams()
+    await runCommand(Cmd, [])
+    expectOutput(stdout.output, `
+Team         Role
+ ──────────── ────────────
+ enterprise a collaborator
+ enterprise b admin
+ team a       collaborator
+ team b       admin
+`)
```

Comment:
You can use `tsheredoc` here to improve readability, but in this case you will need to apply it to both parameters being compared, because tables in OClif Core are indented with one space in the output:

```suggestion
    expectOutput(tsheredoc(stdout.output), tsheredoc`
      Team         Role
      ──────────── ────────────
      enterprise a collaborator
      enterprise b admin
      team a       collaborator
      team b       admin
    `)
```

Reply:
nice, yeah, i created a separate commit because it needs an `import`, but i went ahead and did it, thanks!